### DISCUSSION TITLE
Validate attachment field that all uploads have completed.

### DISCUSSION THREAD

**Discussion by charlfields** [Budibase/budibase#7919] [Help] [answered]
For the attachment input on the form, is there a way to validate that all the uploads have finished uploading before they can submit the form? When uploading a big file, I am able to submit the form before the attachment has completed the upload, and it does not show up on the submitted form. Is there a way to block submitting the form before the upload has completed? Or better yet, is there a way to click a button to start the upload and submit the form once all the uploads have completed at the same time?

**Comment by melohagan** [comment, marked as answer]
Hey @charlfields 

The maximum upload size is 20MB for an attachment. Is that what you mean by big file? 

Looking at the code, I believe it should complete the upload before the field value is updated. In other words, you may be able to use validation on the attachment field.

![Screenshot 2022-09-27 at 11 34 55](https://user-images.githubusercontent.com/101575380/192503709-c528a5d4-0635-4b72-a125-ad54964b974d.png)

**Validation form action in your submit button**
![Screenshot 2022-09-27 at 11 35 42](https://user-images.githubusercontent.com/101575380/192503837-d33c8510-b596-4364-a94c-c151f5182ddd.png)

**Reply by charlfields** [reply]
Thank you this is very helpful
